.github/workflows/release-drafter.yaml

```diff
@@ -1,17 +0,0 @@
-name: Release Drafter
```

Comment:
I don't think this file should be removed. It has to do with the release of the Iter8 CLI as well.

Reply:
@sriumcp I did not remove this file, I just renamed it. [See here](https://github.com/iter8-tools/iter8/pull/1459/files#diff-5c6af029b7e05efb1bde5bf14595b487ca302b580d7b07deea059ead3d1d8a1a)

I just thought the name was ugly because none of the other workflows have a hyphen in their names